djpyfs/djpyfs.py

```diff
@@ -10,35 +10,28 @@
 task can garbage-collect those objects. 
 
 '''
+from __future__ import absolute_import
+from future.builtins import str
```

Comment:
It just happened to do what I needed, and I preferred the option of writing the code as close to Py3 as possible. I see now that OEP-7 indicates a preference for `six`, and the changes ended up being so minimal that I can update that trivially.